docs/public-networks/reference/api/index.md

```diff
@@ -5408,7 +5408,11 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"eth_uninstallFilter","params":["
 
 ## `MINER` methods
 
-The `MINER` API methods allow you to control the node’s mining operation.
+The `MINER` API methods allow you to control: 
+
+* The node’s mining operation.
```

Comment:
"The" article not needed. 

Usually, inanimate objects don't take a possessive.